redcap_hooks.php

```diff
@@ -79,14 +79,14 @@ function redcap_custom_verify_username($username) {
 
 
 // redcap_data_entry_form (REDCap >= 5.11.0)
-function redcap_data_entry_form($project_id, $record, $instrument, $event_id, $group_id) {
+function redcap_data_entry_form($project_id, $record, $instrument, $event_id, $group_id, $repeat_instance) {
```

Comment:
The REDCap document shows the `$repeat_instance` param with a default, but I wasn't sure whether I should set it here.